velox/functions/prestosql/ArraySort.cpp

```diff
@@ -193,7 +209,11 @@ class ArraySortFunction : public exec::VectorFunction {
   /// and 'offsets' vectors that control where output arrays start and end
   /// remain the same in the output ArrayVector.
 
-  explicit ArraySortFunction(bool ascending) : ascending_{ascending} {}
+  explicit ArraySortFunction(
+      bool ascending,
+      bool throwOnNestedNullCompare = true)
```

Comment:
Do we need the default value here? If not, let's remove it. It is easier to reason about the code when default values are not present.

Reply:
Got it, will remove it. I added the default value here to emphasize that the `array_sort` checks nested nulls by default.